doc/modules/grid_search.rst

```diff
@@ -272,10 +628,9 @@ utility function.
 Parallelism
 -----------
 
-:class:`GridSearchCV` and :class:`RandomizedSearchCV` evaluate each parameter
-setting independently.  Computations can be run in parallel if your OS
-supports it, by using the keyword ``n_jobs=-1``. See function signature for
-more details.
+The parameter search tools evaluate each parameter setting independently.
+Computations can be run in parallel if your OS supports it, by using the
+keyword ``n_jobs=-1``. See function signature for more details.
```

Comment:
link to the glossary?

Reply:
FYI I think this user guide is really really good. This is such an annoyingly difficult thing to describe and I think you're doing a great job! I'm mostly nitpicking.

Reply:
Thanks! :)